packages/loot-core/src/server/encryption/encryption-internals.web.ts

```diff
@@ -10,24 +10,14 @@ function browserAlgorithmName(name) {
   }
 }
 
-export async function sha256String(str) {
-  // @ts-expect-error TextEncoder might not accept an argument
-  const inputBuffer = new TextEncoder('utf-8').encode(str).buffer;
-  const buffer = await crypto.subtle.digest('sha-256', inputBuffer);
```

Comment:
Removing Jest from all packages affects types in a way I don't quite understand which breaks this function (I think it's related to Typescript versions). The function is unused so I just removed it. Technically this change could be done later but this PR is small so I included it here